sw/device/tests/clkmgr_jitter_frequency_test.c

```diff
@@ -36,6 +50,51 @@ enum {
 static dif_clkmgr_t clkmgr;
 static dif_pwrmgr_t pwrmgr;
 
+// Test with thresholds for jitter enabled expecting no failures, and then
+// with thresholds for jitter disabled expecting failures.
+static void test_clock_frequencies_with_jitter_enabled(uint32_t delay_micros) {
+  LOG_INFO("Testing frequencies with jitter enabled");
+  CHECK_STATUS_OK(clkmgr_testutils_enable_clock_counts_with_expected_thresholds(
+      &clkmgr, /*jitter_enabled=*/true, /*external_clk=*/false,
+      /*low_speed=*/false));
+  busy_spin_micros(delay_micros);
+  // This checks there are no errors.
+  CHECK_STATUS_OK(clkmgr_testutils_check_measurement_counts(&clkmgr));
+  CHECK_STATUS_OK(clkmgr_testutils_disable_clock_counts(&clkmgr));
+  if (kDeviceType == kDeviceSimDV || kDeviceType == kDeviceSilicon) {
```

Comment:
IIRC, you added a print statement in earlgrey_1.0.0 to notify the user that we don't check this on FPGA. This would probably be useful here as well.

Reply:
Done.

Reply:
Thanks